en/micro-integrator/docs/references/connectors/salesforce-rest-connector/sf-rest-inbound-endpoint-configuration.md

```diff
@@ -0,0 +1,131 @@
+* Testing the PushTopic Channel.
+* Reset Security Token.
+ 
+## Create a custom object or object in Salesforce.
+
+As first step you need to [create a custom object in Salesforce](https://developer.salesforce.com/docs/atlas.en-us.202.0.api_streaming.meta/api_streaming/create_object.htm). In this scenario we use the `Account` object to store the records.
+ 
+## Creating a PushTopic  
+
+The [PushTopic](https://developer.salesforce.com/docs/atlas.en-us.202.0.api_streaming.meta/api_streaming/create_a_pushtopic.htm) record contains a SOQL query. Event notifications are generated for updates that match the query. Alternatively, you can also use Workbench to create a PushTopic. In this sample we using Salesforce Developer Console to create a Push Topic.
+
+1. **Login** to the **Salesforce Account**. Navigate to the top right corner of the **Home page** and click the **Setup** icon. Then select **Developer Console**.
+
+   <img src="../../../../assets/img/connectors/open-the-developer-console.png" title="Open the Developer Console." width="800" alt="Open the Developer Console."/>
+
+2. After populating the Developer console, click **Debug** -> Open **Execute Anonymous Window**.   
+   
+   <img src="../../../../assets/img/connectors/salesforce-inboundEP-execute-anonymous-window.png" title="Open the Anonymous Window." width="800" alt="Open the Anonymous Window."/>
+   
+3. Add the following entry in the **Enter Apex Code** window and click **Execute**.   
+   
+   <img src="../../../../assets/img/connectors/salesforce-inboundEP-enter-apex-code.png" title="Enter Apex code." width="800" alt="Enter Apex code."/> 
+   
+   ```
+   PushTopic pushTopic = new PushTopic();
+   pushTopic.Name = 'Account';
+   pushTopic.Query = 'SELECT Id, Name FROM Account';
+   pushTopic.ApiVersion = 37.0;
+   pushTopic.NotifyForOperationCreate = true;
+   pushTopic.NotifyForOperationUpdate = true;
+   pushTopic.NotifyForOperationUndelete = true;
+   pushTopic.NotifyForOperationDelete = true;
+   pushTopic.NotifyForFields = 'Referenced';
+   insert pushTopic;
+   ```
+   We are essentially creating a SOQL query with a few extra parameters that watch for changes in a specified object. If the Push Topic is executed successfully then Salesforce is ready to post notification to WSO2 Salesforce Inbound Endpoint, if any changes are made in the Account object in Salesforce. This is because the below Push Topic has been created for Salesforce's Account object. 
+   
+## Subscribing to the PushTopic Channel
+
+In this step, we need to [subscribe](https://developer.salesforce.com/docs/atlas.en-us.202.0.api_streaming.meta/api_streaming/subscribe_to_pushtopic_channel.htm) to the channel that we created with the PushTopic record in the previous step. For this can be done through the `Workbench`. Workbench is a free, open source, community-supported tool which helps administrators and developers to interact with Salesforce for Data Insert, Update, Upsert, Delete and Export purposes. 
```

Comment:
```suggestion
In this step, we need to [subscribe](https://developer.salesforce.com/docs/atlas.en-us.202.0.api_streaming.meta/api_streaming/subscribe_to_pushtopic_channel.htm) to the channel that we created with the PushTopic record in the previous step. For this can be done through the `Workbench`. Workbench is a free, open source, community-supported tool that helps administrators and developers to interact with Salesforce for Data Insert, Update, Upsert, Delete, and Export purposes. 
```

Reply:
Updated.